iron/distribution.yaml

```diff
@@ -562,12 +562,10 @@ repositories:
       url: https://github.com/Azure/azure-iot-sdk-c.git
       version: main
     release:
-      packages:
-      - azure_iot_sdk_c
```

Comment:
@Timple Just to confirm, should this be removed or renamed with dashes instead of underscores?

Reply:
This is auto generated by bloom.

It wasn't there before with the dashes. With the underscores it popped up.

Since this is now in line with before, I'm assuming this will work just like before